etcdserver/auth/auth.go

```diff
@@ -448,29 +443,33 @@ func (s *store) DisableAuth() error {
 // is called and returns a new User with these modifications applied. Think of
 // all Users as immutable sets of data. Merge allows you to perform the set
 // operations (desired grants and revokes) atomically
-func (u User) merge(n User) (User, error) {
+func (s passwordStore) merge(ou, nu User) (User, error) {
```

Comment:
@xiang90 Just changed this as a method to satisfy `PasswordStore` interface to make it not as `store` method.

PTAL.